views/poll-view/src/components/AnswerCard/AnswerCard.tsx

```
@@ -54,18 +54,15 @@ export default function AnswerCard(props: {
-              <j-flex gap="300" a="center" className={styles.voters}>
-                {votes.length > 0 && (
-                  <j-flex a="center">
-                    {votes.map((vote, i) => (
-                      <Avatar did={vote.author} size="xs" style={{ marginLeft: i > 0 ? -10 : 0 }} />
-                    ))}
-                  </j-flex>
-                )}
+              <div className={styles.voters}>
+                {votes.length > 0 &&
+                  votes.map((vote, i) => (
+                    <Avatar did={vote.author} size="xs" style={{ marginLeft: i > 0 ? -10 : 0 }} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)